### DISCUSSION TITLE
[Docker][Podman] ENOTFOUND undefined:undefined

### DISCUSSION THREAD

**Discussion by assasen11** [requarks/wiki#6070] [Help / Questions]
I am have so many issues getting docker-compose to start. The wiki is having a lot of issues connecting to the Database. I can confirm that the Postgres seems to be generating the files it needs.

![Screenshot from 2023-01-19 19-42-56](https://user-images.githubusercontent.com/2252890/213593708-5bde7297-a3ca-47fd-80df-8bbdd19fdf5b.png)

I am running out of ideas. I have checked every other post i could find regarding this. I was able to connect to the Postgres container and confirm the database is there. I also know it is running with Podman but it should work without an issue. Any ideas what else I could be missing? I can force test it with docker-compose if i really have to.

Thanks in advanced.

Update01:

Installed docker-compose on a similar Alma Linux Image and it started right up. Guess this is an incompatibility with Podman and Podman-compose

**Comment by BSGMG18** [comment]
Hey, i spent the whole yesterday night to get this going. Seems like i had a similar issue.
Could u please tell me, what your environment variables for your wikijs-container and your postgres-container are?

**Reply by assasen11** [reply]
So my En is as follows. Mostly Default.

![image](https://user-images.githubusercontent.com/2252890/213791759-46ffa726-4206-455e-aeca-57861e21ea84.png)
